druntime/src/core/demangle.d

```diff
@@ -2783,3 +2788,86 @@ extern (C) private
         errno = err;
     }
 }
+
+private struct ManglingFlagInfo
+{
+    /// The flag value to use
+    ushort flag;
+
+    /// Human-readable representation
+    string value;
+}
+
+private enum TypeCtor : ushort {
+    None      = 0,
+    //// 'x'
+    Const     = (1 << 1),
+    /// 'y'
+    Immutable = (1 << 2),
+    /// 'O'
+    Shared    = (1 << 3),
+    ///
+    InOut     = (1 << 4),
+}
+
+///
```

Comment:
Empty comment

Reply:
Is documentation generated for `private` symbols? And why add this `private` symbol to ddoc but not the others?